### PR TITLE
Nc news 2

### PR DESCRIPTION
**Pull request by crissu91**
task 2 completed

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+describe("/api test suite", () => {
+    test("200: should respond with an okay message", () => {
+    return request(app)
+    .get('/api')
+    .expect(200)  
+    .then(({body}) => {
+        expect(body.msg).toBe("Everything is working fine!")
+        })
+    })
+});
```
Comment (Dutton1990) [inline __tests__/app.test.js:24]: The ticket hasn't asked for this behaviour above, you will have to change this for your next task as we have functionality in mind for the /api endpoint!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+describe("GET api/topics", () => {
+    test("with a status 200, it should return all the topics" , () => {
+    return request(app)
+    .get('/api/topics')
+    .expect(200)
+    .then(({body}) => {
+        expect(body.topics).toBeInstanceOf(Array)
+        expect(body.topics.length).toBe(3)
```
Comment (Dutton1990) [inline __tests__/app.test.js:33]: Nice to have a length check, important!

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    })
+    test("400: responds with bad request for an invalid sort_by", () =>{
+    return request(app)
+    .get('/api/topics?sort_by=kiwi')
+    .expect(400)
+    .then(({ body }) => {
+        expect (body.msg).toBe("Bad request");
+        })
+    })
+});
```
Comment (Dutton1990) [inline __tests__/app.test.js:60]: The ticket hasn't asked for this to implemented - please just stick to what the ticket is asking for 😄 

Looks good though!

**Review by Dutton1990 — APPROVED** [review]
Nice job Cristina! Some extra code that wasn't totally necessary but it's looking good! You can merge and continue 😄

File: __tests__/app.test.js (hunk 3 above)
Comment (crissu91) [reply __tests__/app.test.js:60]: Thanks! I though it asked us to tests for the errors we might get :|

**Pull request merged**